src/js/claims-status/containers/AppealsV2StatusPage.jsx

```diff
@@ -43,15 +51,34 @@ const AppealsV2StatusPage = ({ appeal, fullName }) => {
     !hideDocketStatusTypes.includes(status.type) &&
     !hideDocketAppealTypes.includes(appealType);
 
+  const filteredAlerts = alerts.filter(a => a.type !== ALERT_TYPES.cavcOption);
+  const afterNextAlerts = (
+    <div>
+      {alerts
+        .filter(a => a.type === ALERT_TYPES.cavcOption)
+        .map((a, i) => {
```

Comment:
I decided to stick with the overkill method because
1. I had it all written before it occurred to me that we're only looking for one alert type and there will only be one _of_ that type
2. The claims and appeals world seems pretty nebulous, and we may want to add more alerts after the "What happens next?" section later
    - I talked with @gnakm and she agreed to leave it future-proofed because things may change

That said, it's totally unnecessary and isn't hard to do later should the need arise, so I'm okay with nixing it for now.

Reply:
Thanks for the helpful context @cvalarida, sounds good.